.github/workflows/pull_requests.yaml

```diff
@@ -0,0 +1,45 @@
+name: pull_requests
+
+on:
+  pull-request:
+    branches: [main]
+
+jobs:
+  prettier:
+    name: Prettier Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - name: Install dependencies and run prettier-check
```

Comment:
We don't need this `name` since it's the only thing we do in this job (see `name` on line 9).